lib/rails/generators/alchemy/install/files/application.html.erb

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%= @page.language_code %>">
```

Comment:
You're right it could be different, but is it a real life scenario? I am wondering because even if you render a german page and on that page you render elements from an english page isn't always the underlying page deciding?

Am I missing something else / are you aware of a different scenario?